apps/ledger-live-desktop/src/main/setup.js

```diff
@@ -38,13 +38,15 @@ ipcMain.on("updater", (e, type) => {
   updater(type);
 });
 
-ipcMain.handle("save-logs", async (event, path: { canceled: boolean, filePath: string }) =>
-  Promise.resolve().then(
-    () =>
-      !path.canceled &&
-      path.filePath &&
-      fsWriteFile(path.filePath, JSON.stringify(loggerTransport.logs)),
-  ),
+ipcMain.handle(
+  "save-logs",
+  async (event, path: { canceled: boolean, filePath: string }, experimentalLogs: string | null) =>
+    Promise.resolve().then(
+      () =>
+        !path.canceled &&
+        path.filePath &&
+        fsWriteFile(path.filePath, experimentalLogs || JSON.stringify(loggerTransport.logs)),
+    ),
```

Comment:
```suggestion
  async (event, path: { canceled: boolean, filePath: string }, experimentalLogs: string | null) => !path.canceled && path.filePath && fsWriteFile(path.filePath, experimentalLogs || JSON.stringify(loggerTransport.logs))
```

The returned value of an async function is always a Promise, `Promise.resolve` should not be needed

Reply:
i can sure rewrite it but i've not changed what's in production here. i can sure take a look